sem1/task2.py

```diff
@@ -0,0 +1,7 @@
+"""Input - total number of origami (s), output - how much each of 3 kids made (Sergio = Peter, Kate = 2*(Sergio+Peter))"""
+
+s = int(input('Enter total number (multiple of 4): '))
+if s % 4 == 0:
+    print(f'Peter made: {int(s / 4)}, Kate made: {int(s / 2)}, Sergio made: {int(s / 4)}.')
```

Comment:
У вас две операции повторяются. Для этого существуют переменные.
```suggestion
    print(f'Peter made: {s // 4)}, Kate made: {s // 2)}, Sergio made: {s // 4}.')
```